webui/react/src/pages/TrialDetails/TrialDetailsHeader.tsx

```diff
@@ -32,6 +28,7 @@ const TrialDetailsHeader: React.FC<Props> = ({
   trial,
 }: Props) => {
   const [ isRunningTensorBoard, setIsRunningTensorBoard ] = useState<boolean>(false);
+  const [ trialNeverData, setTrialNeverData ] = useState<boolean>(false);
```

Comment:
do we need this? could the usememo just return the value you want? const trialNeverData = useMemo ...

Reply:
ah it's async so we do. maybe we then don't want it to be useMemo? that could be a useEffect I think

Reply:
I wasn't sure the right way to get the async value, and have it be usable inside `headerOptions`